jcl/src/openj9.criu/share/classes/org/eclipse/openj9/criu/CRIUSupport.java

```diff
@@ -29,6 +29,14 @@
  */
 public final class CRIUSupport {
 	
+	static {
+		try {
+			System.loadLibrary("j9criu29"); //$NON-NLS-1$
+		} catch (UnsatisfiedLinkError e) {
+			System.out.println("Unable to load libcriu");
```

Comment:
Perhaps this should `throw new InternalError(e);`?

Reply:
This should probably also use `AccessController.doPrivileged()`.

Reply:
The class may be loaded from an unprivileged context, in which case it will fail unless `doPrivileged()` is used.

Reply:
How's it look now?